### DISCUSSION TITLE
Github Copilot has stopped working

### DISCUSSION THREAD

**Discussion by tmlewin** [community/community#19099] [Copilot Conversations]
I signed up for the 60 day trial and copilot is broken at the moment . It doesn't give any suggestions.Below is the log

[INFO] [default] [2022-06-21T21:32:50.390Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-06-21T21:32:50.751Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-06-21T21:32:51.016Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 264 ms
[INFO] [streamChoices] [2022-06-21T21:32:51.018Z] solution 0 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5LfmMlYQrH2d6haHKndIwHUDKY0hI}] created: [{1655847170}]
[INFO] [default] [2022-06-21T21:32:51.908Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-06-21T21:32:52.072Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 164 ms
[INFO] [streamChoices] [2022-06-21T21:32:52.072Z] solution 0 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5LfmNjPQ1Z8hpO9kDnKb8fdeBmaF9}] created: [{1655847171}]
[INFO] [default] [2022-06-21T21:32:53.194Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-06-21T21:32:53.318Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 124 ms
[INFO] [streamChoices] [2022-06-21T21:32:53.319Z] solution 0 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5LfmPTcinSMJq4ZusnriTZyGj5Ec9}] created: [{1655847173}]
[INFO] [default] [2022-06-21T21:32:54.494Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-06-21T21:32:54.643Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 149 ms
[INFO] [streamChoices] [2022-06-21T21:32:54.643Z] solution 0 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5LfmQdIfftlXv5V9h3PFZMiosqlo9}] created: [{1655847174}]
[INFO] [default] [2022-06-21T21:32:55.621Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-06-21T21:32:55.732Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 111 ms
[INFO] [streamChoices] [2022-06-21T21:32:55.733Z] solution 0 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5LfmRzD8yQR7gnWFHWw6pBNxxEPey}] created: [{1655847175}]
[INFO] [default] [2022-06-21T21:32:56.705Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-06-21T21:32:57.001Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-06-21T21:32:57.113Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 112 ms
[INFO] [streamChoices] [2022-06-21T21:32:57.114Z] solution 0 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5LfmSe8xrIEzRut1qpGzyqKvYpljF}] created: [{1655847176}]
[INFO] [default] [2022-06-21T21:33:09.437Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-06-21T21:33:09.675Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 238 ms
[INFO] [streamChoices] [2022-06-21T21:33:09.676Z] solution 0 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5LfmfzanHPISH0pwSHVvilfBtHfcl}] created: [{1655847189}]
[INFO] [default] [2022-06-21T21:33:10.659Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-06-21T21:33:10.781Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 121 ms
[INFO] [streamChoices] [2022-06-21T21:33:10.782Z] solution 0 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5Lfmgo8UE0ukkDfFPtUI9LnxPxEVI}] created: [{1655847190}]
[INFO] [default] [2022-06-21T21:33:11.685Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-06-21T21:33:11.816Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 131 ms
[INFO] [streamChoices] [2022-06-21T21:33:11.817Z] solution 0 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5LfmhTrdSklmZk7jUjhJ7oHEFLT5Z}] created: [{1655847191}]
[INFO] [default] [2022-06-21T21:33:12.699Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-06-21T21:33:12.821Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 122 ms
[INFO] [streamChoices] [2022-06-21T21:33:12.822Z] solution 0 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5LfmilFeaWuxJToo1870VGCdACVfV}] created: [{1655847192}]
[INFO] [default] [2022-06-21T21:33:16.186Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-06-21T21:33:16.422Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 236 ms
[INFO] [streamChoices] [2022-06-21T21:33:16.423Z] solution 0 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5LfmlQ11b3qr8yWRd1G4ym1HpcDnz}] created: [{1655847195}]
[INFO] [default] [2022-06-21T21:33:21.147Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-06-21T21:33:21.425Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 278 ms
[INFO] [streamChoices] [2022-06-21T21:33:21.426Z] solution 0 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5LfmqiSbf7bK5GV1XwNyFCjEKud0V}] created: [{1655847200}]
[INFO] [default] [2022-06-21T21:33:35.752Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-06-21T21:33:35.843Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 90 ms
[INFO] [streamChoices] [2022-06-21T21:33:35.843Z] solution 0 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5Lfn59dunvJdRpkgehcbne4kZPFwb}] created: [{1655847215}]
[INFO] [default] [2022-06-21T21:33:39.153Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-06-21T21:33:39.389Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 236 ms
[INFO] [streamChoices] [2022-06-21T21:33:39.390Z] solution 0 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5Lfn8aQsjqF6FN94ka5wUeGVGL6Dk}] created: [{1655847218}]
[INFO] [default] [2022-06-21T21:34:08.611Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-06-21T21:34:08.802Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 190 ms
[INFO] [streamChoices] [2022-06-21T21:34:08.802Z] solution 0 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5LfncnEiQ38UGMCK6ZlaSTCOS9hxX}] created: [{1655847248}]
[INFO] [default] [2022-06-21T21:34:09.309Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-06-21T21:34:09.408Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 98 ms
[INFO] [streamChoices] [2022-06-21T21:34:09.408Z] solution 0 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5LfndZ3iBAvRMNzdNJYOkSxBXDP7N}] created: [{1655847249}]
[INFO] [default] [2022-06-21T21:34:10.385Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-06-21T21:34:10.693Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 308 ms
[INFO] [streamChoices] [2022-06-21T21:34:10.694Z] solution 0 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5LfneJheFYDSU4GX9gZssZvnzsVz0}] created: [{1655847250}]
[INFO] [default] [2022-06-21T21:34:16.052Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-06-21T21:34:16.567Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-06-21T21:34:16.783Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-06-21T21:34:16.883Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 100 ms
[INFO] [streamChoices] [2022-06-21T21:34:16.884Z] solution 0 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5LfnklhyPiXacg9ZDmubZcPqalKCx}] created: [{1655847256}]
[INFO] [default] [2022-06-21T21:34:20.713Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-06-21T21:34:20.953Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 240 ms
[INFO] [streamChoices] [2022-06-21T21:34:20.953Z] solution 0 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5LfnoX4NmTejaC71IJcUtUulRnbC4}] created: [{1655847260}]
[INFO] [default] [2022-06-21T21:34:32.986Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-06-21T21:34:33.405Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-06-21T21:34:33.550Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 145 ms
[INFO] [streamChoices] [2022-06-21T21:34:33.550Z] solution 0 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5Lfo1HZOHkJ7p9palahBKzTlFMG2X}] created: [{1655847273}]
[INFO] [default] [2022-06-21T21:34:33.986Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-06-21T21:34:34.087Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 101 ms
[INFO] [streamChoices] [2022-06-21T21:34:34.088Z] solution 1 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5Lfo15Wdb6cG6KHBEbxrgXy0YVwWx}] created: [{1655847273}]
[INFO] [streamChoices] [2022-06-21T21:34:34.089Z] solution 0 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5Lfo15Wdb6cG6KHBEbxrgXy0YVwWx}] created: [{1655847273}]
[INFO] [streamChoices] [2022-06-21T21:34:34.089Z] solution 2 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5Lfo15Wdb6cG6KHBEbxrgXy0YVwWx}] created: [{1655847273}]
[INFO] [default] [2022-06-21T21:34:35.071Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-06-21T21:34:35.299Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 228 ms
[INFO] [streamChoices] [2022-06-21T21:34:35.300Z] solution 0 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5Lfo2UgSHQgR5BXlxenLBFm4WK3fT}] created: [{1655847274}]
[INFO] [streamChoices] [2022-06-21T21:34:35.300Z] solution 2 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5Lfo2UgSHQgR5BXlxenLBFm4WK3fT}] created: [{1655847274}]
[INFO] [streamChoices] [2022-06-21T21:34:35.300Z] solution 1 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5Lfo2UgSHQgR5BXlxenLBFm4WK3fT}] created: [{1655847274}]
[INFO] [default] [2022-06-21T21:34:37.950Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-06-21T21:34:38.172Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 222 ms
[INFO] [streamChoices] [2022-06-21T21:34:38.173Z] solution 2 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5Lfo5GKdHX4DDMIWqwGYadIK44gCt}] created: [{1655847277}]
[INFO] [streamChoices] [2022-06-21T21:34:38.173Z] solution 1 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5Lfo5GKdHX4DDMIWqwGYadIK44gCt}] created: [{1655847277}]
[INFO] [streamChoices] [2022-06-21T21:34:38.174Z] solution 0 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5Lfo5GKdHX4DDMIWqwGYadIK44gCt}] created: [{1655847277}]
[INFO] [default] [2022-06-21T21:34:39.141Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-06-21T21:34:39.296Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 155 ms
[INFO] [streamChoices] [2022-06-21T21:34:39.297Z] solution 1 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5Lfo6IlwPFATG77Zxk6BpdECxW6x2}] created: [{1655847278}]
[INFO] [streamChoices] [2022-06-21T21:34:39.297Z] solution 2 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5Lfo6IlwPFATG77Zxk6BpdECxW6x2}] created: [{1655847278}]
[INFO] [streamChoices] [2022-06-21T21:34:39.297Z] solution 0 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5Lfo6IlwPFATG77Zxk6BpdECxW6x2}] created: [{1655847278}]
[INFO] [default] [2022-06-21T21:34:40.085Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2022-06-21T21:34:40.234Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 149 ms
[INFO] [streamChoices] [2022-06-21T21:34:40.234Z] solution 1 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5Lfo7ksOOfPL2Dis2rFmV299eO8wr}] created: [{1655847279}]
[INFO] [streamChoices] [2022-06-21T21:34:40.235Z] solution 0 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5Lfo7ksOOfPL2Dis2rFmV299eO8wr}] created: [{1655847279}]
[INFO] [streamChoices] [2022-06-21T21:34:40.235Z] solution 2 returned. finish reason: ["stop"] finishOffset: [undefined] completionId: [{cmpl-5Lfo7ksOOfPL2Dis2rFmV299eO8wr}] created: [{1655847279}]

**Comment by davecheney** [comment]
It looks like copilot is working; try turning off "exclude suggestions which may match public code" in your copilot settings. You should close and reopen your editor after doing this to ensure the setting takes effect immediately.

**Reply by tmlewin** [reply]
should i allow it or disable it?

**Comment by juliancox** [comment]
I'm having the same issue,  I can see the suggestions coming through but the GhostText is not displayed in the code page. 
Version: 1.68.1 (Universal)
Commit: 30d9c6cd9483b2cc586687151bcbcd635f373630
Date: 2022-06-14T12:52:13.188Z (1 wk ago)
Electron: 17.4.7
Chromium: 98.0.4758.141
Node.js: 16.13.0
V8: 9.8.177.13-electron.0
OS: Darwin arm64 21.5.0

**Reply by davecheney** [reply]
@juliancox my advice is the same as above

**Reply by juliancox** [reply]
Didn't make any difference for me.  But interestingly seems to be working for me with files with .mjs extension but not .js extension
